app/src/main/java/com/example/bime/classes/OnSwipeListener.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.bime.classes
+
+import android.content.Context
+import android.view.GestureDetector
+import android.view.GestureDetector.SimpleOnGestureListener
+import android.view.MotionEvent
+import android.view.View
+import kotlin.math.abs
+
+
+open class OnSwipeListener(context: Context?) : View.OnTouchListener {
+    private val gestureDetector: GestureDetector
+
+    init {
+        gestureDetector = GestureDetector(context, GestureListener())
+    }
+
+    override fun onTouch(v: View?, event: MotionEvent?): Boolean {
+        return gestureDetector.onTouchEvent(event)
+    }
+
+    private inner class GestureListener : SimpleOnGestureListener() {
+
+        override fun onDown(e: MotionEvent): Boolean {
+            return true
+        }
+
+        override fun onFling(
+            e1: MotionEvent,
+            e2: MotionEvent,
+            velocityX: Float,
+            velocityY: Float
+        ): Boolean {
+
+            val SWIPE_VELOCITY_THRESHOLD = 40
```

Comment:
change to
            val SWIPE_VELOCITY_THRESHOLD = 20
            val SWIPE_THRESHOLD = 100